python/az/aro/azext_aro/custom.py

```diff
@@ -15,7 +15,7 @@
 from msrest.exceptions import HttpOperationError
 from knack.log import get_logger
 
-import azext_aro.vendored_sdks.azure.mgmt.redhatopenshift.v2020_04_30.models as openshiftcluster
+import azext_aro.vendored_sdks.azure.mgmt.redhatopenshift.v2021_09_01_preview.models as openshiftcluster
```

Comment:
testing with preview API. Is there a better way to do this?

Reply:
I was doing `curl` requests to the RP while working on disk encryption, but it requires you to manually create SP for the cluster, assign permissions, etc (normally CLI handles all this). Example here: https://github.com/Azure/ARO-RP/pull/1569

Reply:
Align with @Makdaam  for CLI changes. He is enabling preview api overall.